server/app/models/fusor/deployment.rb

```diff
@@ -12,7 +12,6 @@
 
 module Fusor
   class Deployment < ActiveRecord::Base
-    serialize :rhev_params, Hash
     serialize :cfme_params, Hash
     serialize :openstack_params, Hash
```

Comment:
I didn't remove the cfme and the openstack ones because I was going to do those as another PR.